extmod/modbluetooth.c

```diff
@@ -69,6 +69,12 @@ typedef struct {
     ringbuf_t ringbuf;
 } mp_obj_bluetooth_ble_t;
 
+STATIC void bluetooth_handle_uninitialized(void) {
```

Comment:
I'd suggest renaming this to something like `bluetooth_check_enabled()`, to match the "enabled" in `mp_bluetooth_is_enabled`

Reply:
Done.